.eslintrc.js

```diff
@@ -1,6 +1,9 @@
-module.exports = {
+export default {
   extends: ['eslint:recommended', 'plugin:@typescript-eslint/recommended'],
   parser: '@typescript-eslint/parser',
   plugins: ['@typescript-eslint'],
   root: true,
+  rules: {
+    '@typescript-eslint/no-explicit-any': 'off',
+  },
 };
```

Comment:
I don't see the point in this eslintrc being a js file. Consider moving to a .eslintrc.json?